ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/CV_PATHFINDER_TESTS.yml

```diff
@@ -85,6 +98,76 @@ wan_edge:
           wan_circuit_id: S512
           set_default_route: true
           ip: dhcp
+  # Testing HA and disabling HA
+  node_groups:
+    # SITE_HA_ENABLED
```

Comment:
I feel it would be more natural with the HA toggle under the node-group here. Similar to how we do for mlag: false.